l10n/en/products/vpn/shared.ftl

```diff
@@ -25,11 +25,20 @@ vpn-shared-features-servers = { $servers }+ servers in { $countries }+ countries
 
 vpn-shared-features-bandwidth = No bandwidth restrictions
 vpn-shared-features-activity = No logging of your network activity
+vpn-shared-features-activity-logs = No online activity logs now or ever
```

Comment:
Ported over items from the [`products/vpn/platforms/shared.ftl`](https://github.com/mozilla/bedrock/blob/10032-vpn-platform-page-updates/l10n/en/products/vpn/platforms/shared.ftl) file that did not exist in the root shared FTL file.

Reply:
This is very simular to the string above `No logging of your network activity`. Rather than adding more translations for essentially the same sentance, I'd suggest talking with Raphael to see if there are strings we could standardize here across pages?

Reply:
@alexgibson Who do we need to sync up with for this? I'm happy to make the switch if there's no SEO-related objections.

Reply:
I'd suggest talking to Raphael on this. If there are obvious keywords that add value then it makes sense, but for some of these strings they are already very similar, and we should avoid creating more work for localizers than necessary. I'll defer to what you ultimately decide here.